homeassistant/components/person/__init__.py

```diff
@@ -318,6 +320,8 @@ def state_attributes(self):
             data[ATTR_LATITUDE] = round(self._latitude, 5)
         if self._longitude is not None:
             data[ATTR_LONGITUDE] = round(self._longitude, 5)
+        if self._gps_accuracy is not None:
+            data[ATTR_GPS_ACCURACY] = round(self._gps_accuracy, 1)
```

Comment:
We should remove all the rounding of all the attributes, we should just present the state data as-is. (also from the other attributes)